Python/Automated_Noise_Removal/AutomatedNoiseRemoval.py

```diff
@@ -0,0 +1,155 @@
+"""
+
+OPEN-SOURCE SCRIPT FOR AUTOMATED NOISE REMOVAL USING PYTHON.
+
+SOFTWARE/LIBRARY REQUIRED TO RUN THIS SCRIPT:
+	1. SoX - ( http://sox.sourceforge.net/ )
+	2. pydub - ( https://pypi.org/project/pydub/ )
+
+CREATED AND SUBMITTED BY RAHUL RANA @owlrana during GSSoC'21 PERIOD for https://github.com/HarshCasper/Rotten-Scripts
+
+If you have any questions/queries related to this script you can contact me at: me.rahulrana@outlook.com
+
+"""
+
+
+import sys
+import os
+import glob
+import subprocess
+from pydub import AudioSegment
+
+
+MYSOX_PATH = '"C:\\Users\\Rahul Rana\\OneDrive - Microsoft Student Ambassadors\\Azure Applications\\sox\\sox.exe"' # IS DIFFERENT FOR EACH SYSTEM!! NEED TO INSTALL SOX FIRST!!
```

Comment:
Remember to add links to SOX.

And python hates inline comments, please put the comment in `l22`